### PR TITLE
rename error analysis to debugml and fix breadcrumb

### PR DESCRIPTION
**Pull request by imatiach-msft**
- renames error analysis to DebugML
- fixes breadcrumb issue: https://github.com/microsoft/responsible-ai-widgets/issues/209

### CODE REVIEW

File: libs/error-analysis/src/lib/ErrorAnalysisDashboard/Controls/MainMenu/MainMenu.tsx [hunk 1]
```diff
@@ -91,11 +91,11 @@ export class MainMenu extends React.PureComponent<IMainMenuProps> {
...
           commandBarButtonAs: (): any => (
             <Label styles={labelStyle}>
-              {localization.ErrorAnalysis.MainMenu.errorDetectorLabel}
+              {localization.ErrorAnalysis.MainMenu.errorExplorerLabel}
             </Label>
           ),
-          key: "errorDetectorLabel",
-          text: "Error Detector Label"
+          key: "errorExplorerLabel",
+          text: "Error explorer label"
```
Comment (romanlutz) [inline libs/error-analysis/src/lib/ErrorAnalysisDashboard/Controls/MainMenu/MainMenu.tsx:98]: this should be localized I think (?)

**Review by romanlutz — APPROVED** [review]

File: libs/error-analysis/src/lib/ErrorAnalysisDashboard/Controls/MainMenu/MainMenu.tsx (hunk 1 above)
Comment (imatiach-msft) [reply libs/error-analysis/src/lib/ErrorAnalysisDashboard/Controls/MainMenu/MainMenu.tsx:98]: it's not shown anywhere I believe?

Comment (romanlutz) [reply libs/error-analysis/src/lib/ErrorAnalysisDashboard/Controls/MainMenu/MainMenu.tsx:98]: ... in which case: why is there a label?

Comment (imatiach-msft) [reply libs/error-analysis/src/lib/ErrorAnalysisDashboard/Controls/MainMenu/MainMenu.tsx:98]: the commandBarButtonAs overrides it

**Pull request merged**